### DISCUSSION TITLE
Maelstrom v0.6

### DISCUSSION THREAD

**Discussion by sleepeybunney** [sleepeybunney/maelstrom#62] [General]
* New Feature:
  * Randomised Doomtrain Puzzle
* Random Bosses:
  * New option to adjust bosses' stats to match the encounter they're replacing
  * New option to restrict where certain endgame bosses may appear
  * Added to the pool:
    * Ultimecia
    * Griever
    * Grievermecia
* Other Changes:
  * Added animations for cut spells
  * Fixed seed not appearing in the spoiler file
  * Fixed Tonberry King not spawning
  * Fixed Tonberry King replacement not spawning
  * Fixed Tonberry King replacement spawning but being immortal
  * Fixed Tonberry King replacement not showing up for their other fights when duplicate bosses are enabled

<hr /><em>This discussion was created from the release <a href='https://github.com/sleepeybunney/maelstrom/releases/tag/v0.6'>Maelstrom v0.6</a>.</em>

**Comment by JayH2971** [comment]
YES! YES! Thank you!! Time to roll some more seeds!!

**Comment by Cerberusyuri** [comment]
For some reason i can't get the randomizer to work with either of the FFVIII games.

**Reply by sleepeybunney** [reply]
Make sure you've extracted all the Maelstrom files from the zip file - Windows lets you open it like a folder & run things straight out of it but it doesn't really work.

Some people have trouble modifying the game if it's installed under Program Files & need to move it out.

If you've been using other modding tools make sure they're closed, in case they're sitting on any files. Try rebooting as well.

If none of this works, any more information you can give me would be helpful! Install locations, error messages, what settings you're using, etc.

**Reply by Cerberusyuri** [reply]
a while after writing the previous post i could get it to work, not sure where the problem was but the only thing i changed this time aside from keeping keeping the 4 basics for G.F. i kept your modding tool open while starting it. not sure if that fixed it or if i just, by some randomness, hit the exact same skills for the G.F.  but well it works now ^^ so still thanks for the answer^^
